main.js

```diff
@@ -418,13 +417,9 @@ function initHomeegram(homeegram) {
     adapter.getObject(nodeId, (err, obj) => {
         if (!err && obj) {
             adapter.extendObject(nodeId, {
-                type: 'state',
+                type: 'device',
```

Comment:
why you changed this?

Reply:
having a state with button-functionality with states beneath it was confusing for me when i started to use this adapter. seems more "logical" to me this way. now only .active / .play  / .playing are states and the homeegramm-id is the device housing those.
besides, the button didn't even work.. 😝